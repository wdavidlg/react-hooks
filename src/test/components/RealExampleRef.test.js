const { shallow } = require("enzyme")
const { RealExampleRef } = require("../../components/04-useRef/RealExampleRef")

describe('Pruebas con <RealExampleRef/>', () => {
    
    test('debe mostrarse correctamente', () => {
        const wrapper = shallow(<RealExampleRef/>);
        expect(wrapper).toMatchSnapshot();
        
    })

    test('debe de mostrar el componente <MultipleCustomHooks/>', () => {
        const wrapper = shallow(<RealExampleRef/>);
        wrapper.find('button').simulate('click');
        const resp = wrapper.find('MultipleCustomHooks').exists();
        expect(wrapper).toMatchSnapshot();
        expect(resp).toBe(true);
    })
    
    
})
