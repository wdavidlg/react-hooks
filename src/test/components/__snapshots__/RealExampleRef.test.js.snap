// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas con <RealExampleRef/> debe de mostrar el componente <MultipleCustomHooks/> 1`] = `
<Fragment>
  <h1>
    Real Example Ref
  </h1>
  <hr />
  <MultipleCustomHooks />
  <button
    className="btn btn-outline-primary mt-5"
    onClick={[Function]}
  >
    Show-Hide
  </button>
</Fragment>
`;

exports[`Pruebas con <RealExampleRef/> debe mostrarse correctamente 1`] = `
<Fragment>
  <h1>
    Real Example Ref
  </h1>
  <hr />
  <button
    className="btn btn-outline-primary mt-5"
    onClick={[Function]}
  >
    Show-Hide
  </button>
</Fragment>
`;
